src/mbgl/vulkan/context.cpp

```diff
@@ -27,8 +27,20 @@
 #include <algorithm>
 #include <cstring>
 
-#ifndef MLN_VULKAN_DESCRIPTOR_POOL_SIZE
-#define MLN_VULKAN_DESCRIPTOR_POOL_SIZE 30000
+#ifndef MLN_VULKAN_GLOBAL_DESCRIPTOR_POOL_SIZE
+#define MLN_VULKAN_GLOBAL_DESCRIPTOR_POOL_SIZE 3 * 4
```

Comment:
Could you use `constexpr` here? Ideally without all caps.

Reply:
They're used as defines so they can be overridden with build time configs if needed. I don't have a strong preference either way since they're just initial values in resizable containers.

Reply:
I suggest we use `constexpr` until someone comes along that actually wants to set these with a build config.